omod/src/main/java/org/openmrs/module/fhir/swagger/SwaggerSpecificationCreator.java

```diff
@@ -0,0 +1,641 @@
+                    Schema schemaSuccess = new Schema();
+                    schemaSuccess.setRef(getSchemaRef(resourceName));
+                    schemaSuccess.setType(SwaggerDocConstants.OBJECT);
+                    responseSuccess.setSchema(schemaSuccess);
+                    responseMap.put(SwaggerDocConstants.SUCCESS_RESPONSE_CODE, responseSuccess);
+
+                    Response responseError = new Response();
+                    responseError.setDescription(SwaggerDocConstants.ERROR_OCCURRED);
+                    Schema schemaError = new Schema();
+                    schemaError.setRef(getSchemaRef(SwaggerDocConstants.GENERAL_ERROR));
+                    schemaError.setType(SwaggerDocConstants.OBJECT);
+                    responseError.setSchema(schemaError);
+                    Map<String, String> examplesError = new HashMap<String, String>();
+                    examplesError.put(SwaggerDocConstants.CONSUMES_JSON, SwaggerDocConstants.ERROR_PAYLOAD);
+                    responseError.setExamples(examplesError);
+                    responseMap.put(SwaggerDocConstants.ERROR_RESPONSE_CODE, responseError);
+                    everythingOp.setResponses(responseMap);
+
+                    everythingOperationsMap.put(SwaggerDocConstants.POST, everythingOp);
+                    everything.setOperations(everythingOperationsMap);
+                    pathMap.put(pathId, everything);
+                }
+            }
+
+            createDefinition(SwaggerDocConstants.GENERAL_ERROR);
+            createDefinition(SwaggerDocConstants.OPERATION_OUTCOME);
+            createDefinition(SwaggerDocConstants.BUNDLE);
+        }
+        fullPaths.setPaths(pathMap);
+        swaggerSpecification.setPaths(fullPaths);
+    }
+
+    private void createObjectDefinitions() {
+        Definitions definitions = new Definitions();
+        definitions.setDefinitions(definitionMap);
+        swaggerSpecification.setDefinitions(definitions);
+    }
+
+    private void createDefinition(String resourceName) {
+        String definitionName = resourceName;
+        Definition definition = new Definition();
+        definition.setType(SwaggerDocConstants.OBJECT);
+        definitionMap.put(definitionName, definition);
+    }
+
+    /**
+     * @return the swaggerSpecification
+     */
+    public SwaggerSpecification getSwaggerSpecification() {
+        return swaggerSpecification;
+    }
+
+
+    private String createSwaggerSpecification() {
+        String json = "";
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            mapper.configure(SerializationFeature.INDENT_OUTPUT, true);
+            mapper.configure(SerializationFeature.WRITE_NULL_MAP_VALUES, true);
+            mapper.setSerializationInclusion(Include.NON_NULL);
+            mapper.getSerializerProvider().setNullKeySerializer(new NullSerializer());
+            json = mapper.writeValueAsString(swaggerSpecification);
+        } catch (Exception exp) {
```

Comment:
@harsha89 there is nothing to do by catching specific exception. I have remove the print stack trace and added log